WORKSPACE

```diff
@@ -46,6 +37,13 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.38.1/rules_nodejs-0.38.1.tar.gz"],
 )
 
+git_repository(
+    name = "io_bazel_rules_go",
```

Comment:
Please use http_archive as it was before. For an example of http_archive with a specific commit, look at `com_github_atlassian_bazel_tools`